src/forms/dispense-form.component.tsx

```diff
@@ -41,7 +41,11 @@ const DispenseForm: React.FC<DispenseFormProps> = ({
     if (!isSubmitting) {
       setIsSubmitting(true);
       const abortController = new AbortController();
-      saveMedicationDispense(medicationDispensePayload, abortController).then(
+      saveMedicationDispense(
+        medicationDispensePayload,
+        MedicationDispenseStatus.completed,
```

Comment:
we now set the status of medication dispense when saving... this allows us to use the same function to initialize the dispense object for the dispense, pause and close forms... these forms then set the status when saving